testpilot/experiments/tests.py

```diff
@@ -288,7 +288,8 @@ def test_features_list(self):
     def test_me_list(self):
         """/api/me installation listing should include feature flags"""
         result = self.jsonGet('me-list')
-        self.assertEqual(result['installed'][0]['features'], {
+        print(list(result['installed'].values()))
```

Comment:
Nit: Should probably take out this print statement, though it's in a test and won't affect the server